app/scripts/views/mixins/flow-events-mixin.js

```diff
@@ -23,6 +25,38 @@ define(function (require, exports, module) {
         flowBeginTime: flowBegin,
         flowId: flowId
       });
+    },
+
+    events: {
+      'click a': '_clickFlowEventsLink',
+      'click input': '_engageFlowEventsForm',
+      'input input': '_engageFlowEventsForm',
+      'keyup input': '_keyupFlowEventsInput',
+      'submit': '_submitFlowEventsForm'
+    },
+
+    _clickFlowEventsLink (event) {
+      if (event && event.target && event.target.id) {
```

Comment:
Good point, I'll switch to a `data-` attribute to more explicitly express.

The conditional behaviour I see as a feature though, so I'm not sure it makes sense to log an error. Emitting an event should be opt-in, there are links for which we don't want to emit one.